src/timestep.f90

```diff
@@ -87,19 +90,19 @@ subroutine timestep
 
   dt = courant * minval(dti)
 
-  ! Reduce quantities across all MPI tasks
-  call allreduce_mpi('min',dt)
-  call allreduce_mpi('max',cfmax)
```

Comment:
Is there a reason why this was deleted?

Reply:
I mean, `cfmax`

Reply:
It's just been moved further up to line 50 alongside the reduction for `cgrav`. I wanted to put the reduction closer to where the value is computed so that it's clear the MPI `max` operation is related to the local `max()` call. 

Also, in the case of `cgrav`, it's necessary to reduce the value straight away because it's then used in the following lines to compute `dtgrav`.

Reply:
Oops, sorry I missed that. All good!